parallelstone_manager/services/rcon.py

```
@@ -12,7 +12,18 @@ def __init__(self, host: str, port: int, password: str):
         self.request_id = 0
-    
+
+    async def is_connected(self) -> bool:
+        """연결 상태 확인 (소켓 레벨)"""
+        if self.writer is None:
+            return False
+
+        # 소켓이 닫혔는지 확인
+        try:
+            return not self.writer.is_closing()
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)